CHANGELOG.md

```diff
@@ -6,6 +6,62 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.10.0] - Jan, 31, 2022
+
+### Core
+
+#### Added
+
+- [Icons] Thumb icons
+
+#### Fixed
+
+- [Text Input, TextArea] Prevent rounded corners in iOS browsers
+
+### React
+
+#### Added
+
+- [Icons] Thumb icons
+
+#### Changed
+
+- [Tabs] Add support to use Tabs child components in Tabs namespace, for example, Tabs.TabList
+
+#### Fixed
+
+- [Text Input, TextArea] Prevent rounded corners in iOS browsers
```

Comment:
Should it be Text Area, as Text Input is also written as two separate words?

Reply:
I used the same convention as in elsewhere 🤔

Reply:
There seems to three kind of convention for these two: "Time input", "Time Input", "TimeInput" and "Text area", "TextArea" and "Textarea" 🙈

Reply:
TextInput or Text input. TextInput when talking about the specific component and "Text input" when talking about the element in general.

Reply:
- [ ] Fix component naming